R/de_functions.R

```diff
@@ -73,7 +73,8 @@ collapseCellsByType <- function(cm, groups, min.cell.count) {
   names(g1) <- g1.n
   g1[g1 %in% droplevels] <- NA
   g1 <- as.factor(g1)
-  aggr <- Matrix.utils::aggregate.Matrix(cm, g1)
+  cm.clean <- cm %<>% .[rownames(.) %in% names(g1),]
+  aggr <- Matrix.utils::aggregate.Matrix(cm.clean, g1)
```

Comment:
This probably will be conflicting with Peter's changes of the function. Please, pull the latest update from the dev branch and merge it.